### DISCUSSION TITLE
Central place for plugins

### DISCUSSION THREAD

**Discussion by Barina** [searxng/searxng#4067] [Q&A] [answered]
Hellos! 
Do we have a central page for installing & configuring plugins? 
Built-in and\or 3rd party
I can't find any..

**Comment by return42** [comment]
If you ask for the documentation, please have a look here: https://docs.searxng.org/dev/plugins.html

**Comment by Barina** [comment]
Hi return42 thanks for your quick reply!
I've already went through the plugins wiki :) however this is not what I'm after.. 
Take a look at [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted), I'm sure you know.. this is extremely overwhelming but I thought maybe we have something (not necessarily official and not necessarily at that scale lol) similar for searx\ng mainly for 3rd party plugins but includes the officials :)

**Comment by return42** [comment, marked as answer]
There is no such thing as what you expect.

**Reply by Barina** [reply]
Thanks :) hope we will some day

**Comment by dalf** [comment]
Related to https://github.com/searxng/searxng/issues/534
